src/components/Disputes/DisputeCurrentRuling.js

```diff
@@ -55,42 +55,81 @@ function DisputeCurrentRuling({ dispute }) {
-              css={`
-                background: ${color};
-                width: 8px;
-                height: 8px;
-                margin-right: 8px;
-                border-radius: 50%;
-              `}
-            />
+        renderFullLegendItem={({ color, item, index, percentage }) => {
+          // We'll show the juror voting weight hint only if the juror participated in the  current ruling
+          // and has been drafted more than once
+          const showMyVotingWeight = index === 0 && myWeight > 1
 
-            <span
-              key={index}
-              css={`
-                color: ${theme.contentSecondary};
-                width: 220px;
-              `}
-            >
-              {item}
-            </span>
-            <span
+          return (
+            <div
               css={`
-                margin-right: ${1 * GU}px;
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+                width: 100%;
               `}
             >
-              {percentage}%
-            </span>
-            {myDistributionIndex === index && <Tag>You</Tag>}
-          </div>
-        )}
+              <div
+                css={`
+                  display: flex;
+                  align-items: center;
+                `}
+              >
+                <div
+                  css={`
+                    background: ${color};
+                    width: 8px;
+                    height: 8px;
+                    margin-right: 8px;
+                    border-radius: 50%;
+                  `}
+                />
+
+                <span
+                  key={index}
+                  css={`
+                    color: ${theme.contentSecondary};
+                    width: 220px;
+                  `}
+                >
+                  {item}
+                </span>
+                <span
+                  css={`
+                    margin-right: ${1 * GU}px;
+                  `}
+                >
+                  {percentage}%
+                </span>
+                {myDistributionIndex === index && <Tag>You</Tag>}
+              </div>
+              {showMyVotingWeight && (
+                <div
+                  css={`
+                    display: flex;
+                    align-items: center;
+                  `}
+                >
+                  <span
+                    css={`
+                      color: ${theme.help};
+                      margin-right: ${0.5 * GU}px;
+                    `}
+                  >
+                    Your voting weight
+                  </span>
+                  <Help>
+                    The same juror can be drafted multiple times to arbitrate
+                    the same dispute for the same round. When that happens,
+                    their voting weight will be proportional to the number of
+                    times is drafted, as well as the % of ANJ locked in the
```

Comment:
```suggestion
                    times they are drafted, as well as the % of ANJ locked in the
```

I don't follow the second half of this sentence ("as well as the % of ANJ locked in the Active balance"). This is irrelevant to the voting weight, right? It only helps determine how many times they could be drafted.

Reply:
Yeah i think that part was meant as extra information but it should be changed to the `amount` rather than `%` of ANJ, or we could drop it entirely.